lib/lotus/view/rendering/partial_finder.rb

```diff
@@ -42,6 +45,14 @@ def find
         end
 
         protected
+        # @since 0.6.0
+        # @api private
+        def find_cached_template
+          partials = Lotus::View.configuration.partials[[view_template_dir, template_name].join(separator)] ||
```

Comment:
@stevehook This is [Inappropriate Intimacy](http://c2.com/cgi/wiki?InappropriateIntimacy) of `Configuration`. This method knows that partials are stored as a Hash and the structure of keys and values.

It could be opportune to expose `Configuration#partial_for`, we can pass all the needed arguments to this new method and let `find_cached_template` to just invoke it.